src/libraries/Microsoft.PowerFx.Core/Syntax/Visitors/DTypeVisitor.cs

```diff
@@ -5,143 +5,153 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using Microsoft.PowerFx.Core.Binding;
 using Microsoft.PowerFx.Core.Binding.BindInfo;
+using Microsoft.PowerFx.Core.Public.Types;
 using Microsoft.PowerFx.Core.Types;
 using Microsoft.PowerFx.Core.Utils;
 using Microsoft.PowerFx.Syntax;
 using Microsoft.PowerFx.Types;
 
 namespace Microsoft.PowerFx.Core.Syntax.Visitors
 {
-    internal class DTypeVisitor : TexlFunctionalVisitor<DType, DefinedTypeSymbolTable>
+    internal class DTypeVisitor : TexlFunctionalVisitor<DType, INameResolver>
```

Comment:
>DTypeVisitor [](http://example.com/codeflow?start=19&length=12)

Can we add a comment for this class?

is this effectively the binder for types? 
converting a type AST into a DType?

Reply:
yes , will add that comment